frontend/awx/resources/inventories/inventoryHostsPage/InventoryHostForm.tsx

```diff
@@ -133,31 +158,73 @@ export function CreateHost() {
           variables: '---\n',
         }}
       >
-        <HostInputs />
+        <HostInputs edit_mode={false} inventory_host={params.inventory_host} />
       </AwxPageForm>
     </PageLayout>
   );
 }
 
+function useHostParams(mode: 'edit' | 'create'): {
+  id?: string;
+  inventory_type?: string;
+  host_id?: string;
+  group_id?: string;
+  inventory_host: boolean;
+} {
+  const params = useParams<{
+    id: string;
+    inventory_type: string;
+    host_id: string;
+    group_id: string;
+  }>();
+
+  let id = params.id;
+  let host_id = params.host_id;
+
+  let inventory_host = false;
+
+  if (!host_id && mode === 'edit') {
+    // this means the form was opened from host list, which passes host_id as id, so we have to rename it
```

Comment:
We need to remove comments before merging

Reply:
Ok, But I left this for purpose.